composer.json

```diff
@@ -94,7 +94,8 @@
         "auto-scripts": {
             "make cache-warmup": "script",
             "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd",
```

Comment:
Did you mean to leave in the 2 preceding commands?

Right now, the cache is warmed and then immediately cleared. While the second `assets:install` without symlink and relative will copy the assets into the already symlinked locations and that might have unintended side-effects.

Reply:
Can you send a PR to correct this?

Reply:
@codeliner Done in PR #39

Reply:
cool, thx!